CHANGELOG.md

```diff
@@ -1,9 +1,21 @@
 vNext (Month Day, Year)
 -----------------------
 **Breaking Changes**
+- `<sevicename>::from_env()` has been removed (#675). A drop-in replacement is available:
+  1. Add a dependency on `aws-config`
+  2. ```rust
+        let client = <service>>::Client::new(&aws_config::load_from_env().await)
+     ```
+
+- `ProvideRegion` has been moved to `aws_config::meta::region::ProvideRegion`. (#675)
+- `aws_types::region::ChainProvider` has been moved to `aws_config::meta::region::RegionProviderChain` (#675).
 - `ProvideRegion` is now asynchronous. Code that called `provider.region()` must be changed to `provider.region().await`.
-- `<awsservice>::Config::from_env()` is now also asynchronous because it must load a region
-- `<awsservice>::Config::builder()` will **not** load a default region unspecified. A region must be specified directly.
+- `<awsservice>::Config::builder()` will **not** load a default region unspecified. To preserve previous behavior:
```

Comment:
I'm having a hard time grokking "will not load a default region unspecified". What is "unspecified" signaling? Maybe get rid of it?
```suggestion
- `<awsservice>::Config::builder()` will **not** load a default region. To preserve previous behavior:
```